### DISCUSSION TITLE
docfx --version command outputs the wrong version?

### DISCUSSION THREAD

**Discussion by bimCelum** [dotnet/docfx#9670] [Q&A] [answered]
Hi, I currently have docfx version 2.75 installed. Updates via `dotnet update \-g docfx` seem to work fine and indicate the correct version of the tool. However, when I run `docfx \--version` (in cmd as administrator), I consistently get the wrong output of "`2.70.2+`" with a random sequence of numbers and letters after.

![image](https://github.com/dotnet/docfx/assets/20797331/c8efb0c2-aaa9-4e52-aa37-f87b6102e49f)

Is there some older version of docfx hanging around my machine somewhere, screwing with this command? How can I fix that?
OS: Windows 11 professional

**Comment by filzrev** [comment, marked as answer]
What's the result when you run the following commands.

> where docfx

If docfx is installed as .NET global tools.
It' expected to be `%USERPROFILE%\.dotnet\tools\docfx.exe`.

When other exe path are displayed
It need to confirm `PATH` settings.

**Comment by bimCelum** [comment]
Thanks for the reply, filzrev. It works fine now after adapting the PATH variable.
